### PR TITLE
Feature/mygfw aois

### PR DESCRIPTION
**Pull request by edbrett**
## Overview

Removes the old MyGFW and replaces it with the 🆕 MyGFW! Now it is a single page listing your profile in the header with a modal to edit your profile, and your areas of interest below.

## Testing

- go to `/my-gfw`

### CODE REVIEW

File: app/javascript/assets/icons/logout.svg [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<<<<<<< HEAD
```
Comment (blayhem) [inline app/javascript/assets/icons/logout.svg]: Looks like there's a conflict with this one

File: app/javascript/components/analysis/index.js [hunk 2]
```diff
@@ -7,7 +7,6 @@ import reducerRegistry from 'app/registry';
 
 import { setSubscribeSettings } from 'components/modals/subscribe/actions';
 import { setSaveAOISettings } from 'components/modals/save-aoi/actions';
-import { clearActiveArea } from 'providers/areas-provider/actions';
```
Comment (blayhem) [inline app/javascript/components/analysis/index.js]: how is it managing this now?

File: app/javascript/components/aoi-card/component.jsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  fw: 'Create {date} on Forest Watcher',
+  map: 'Created {date} on Map Builder'
+};
+
+const counts = {
+  GLAD: 12352,
+  Fires: 221176
+};
+
+class AoICard extends PureComponent {
```
Comment (blayhem) [inline app/javascript/components/aoi-card/component.jsx]: nice job with this component 👍

File: app/javascript/components/map-menu/components/sections/my-gfw/selectors.js [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
+export const getActiveArea = createSelector(
+  [selectLocation, selectAreas],
+  (location, areas) => {
+    if (isEmpty(areas)) return null;
+
+    return areas.find(a => a.id === location.adm0);
+  }
+);
+
+export const getTags = createSelector([selectAreas], areas => {
```
Comment (blayhem) [inline app/javascript/components/map-menu/components/sections/my-gfw/selectors.js]: good idea moving these to a selectors file!

File: app/javascript/components/modals/save-aoi/component.jsx [hunk 5]
```diff
@@ -8,73 +8,64 @@ import Loader from 'components/ui/loader';
...
+    if (!saving && saving !== prevProps.saving) {
+      this.handleCloseModal();
+    }
+  }
+
   handleCloseModal = () => {
     const { setSaveAOISettings, resetSaveAOI, saving } = this.props;
     if (!saving) {
-      setSaveAOISettings({ open: false });
+      setSaveAOISettings({ open: false, activeAreaId: null });
```
Comment (blayhem) [inline app/javascript/components/modals/save-aoi/component.jsx]: ah I see the clearArea thing, nice

File: app/javascript/pages/mygfw/component.jsx [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+import ShareModal from 'components/modals/share';
+import ProfileModal from 'components/modals/profile';
+
+import AreasProvider from 'providers/areas-provider';
+import CountryDataProvider from 'providers/country-data-provider';
+
+import SaveAOIModal from 'components/modals/save-aoi';
+import UserProfile from './components/user-profile';
+import AreasTable from './components/areas-table';
+import NoAreasImage from './assets/no-areas@2x.png';
```
Comment (blayhem) [inline app/javascript/pages/mygfw/component.jsx]: check the assets, this one looks out of place here

File: app/javascript/assets/icons/logout.svg (hunk 1 above)
Comment (edbrett) [reply app/javascript/assets/icons/logout.svg]: How on earth did the pre commit checks miss that 😕

File: app/javascript/components/analysis/index.js (hunk 2 above)
Comment (edbrett) [reply app/javascript/components/analysis/index.js]: Yes as you saw below.

File: app/javascript/components/map-menu/components/sections/my-gfw/selectors.js (hunk 4 above)
Comment (edbrett) [reply app/javascript/components/map-menu/components/sections/my-gfw/selectors.js]: :)

File: app/javascript/pages/mygfw/component.jsx (hunk 6 above)
Comment (edbrett) [reply app/javascript/pages/mygfw/component.jsx]: Yes, I need fausto to come back, as the export from invision is unusable.

Comment (edbrett) [reply app/javascript/pages/mygfw/component.jsx]: As that is a long way off I am going to leave this for now.

**Pull request merged**